### ISSUE TITLE
Epochs do not update when training

### ISSUE THREAD

**Issue by fennex628** [mozilla/TTS#94]
Hi! I have encountered an issue regarding the updating of training epochs, using train.py and the LJSpeech-1.1 dataset. I am unsure if training is even taking place, due to the lack of updates printed in the terminal.

I have followed the installation instructions in the readme, and have split the metadata.csv file as directed. Upon running the following command from the terminal in the TTS directory:
`sudo python3 train.py --config_path /home/dev/TTS/config.json`
I am presented with the following information in the terminal:

> Using CUDA:  True
 > Number of GPUs:  4
 > Git Hash: e78911b
 > Experiment folder: /home/dev/TTS/output/January-02-2019_05+41PM-TTS-master-e78911b
 > Setting up Audio Processor...
 | > fft size: 2048, hop length: 275, win length: 1102
 | > Audio Processor attributes.
   | > bits:None
   | > sample_rate:22050
   | > num_mels:80
   | > min_level_db:-100
   | > frame_shift_ms:12.5
   | > frame_length_ms:50
   | > ref_level_db:20
   | > num_freq:1025
   | > power:1.5
   | > preemphasis:0.97
   | > griffin_lim_iters:30
   | > signal_norm:True
   | > symmetric_norm:False
   | > mel_fmin:0
   | > mel_fmax:None
   | > max_norm:1.0
   | > clip_norm:True
   | > do_trim_silence:True
   | > n_fft:2048
   | > hop_length:275
   | > win_length:1102
 | > Number of characters : 149
 | > Num output units : 1025

 > Starting a new training
 | > Model has 7083650 parameters
 > Reading LJSpeech from - /home/dev/TTS/datasets/LJSpeech-1.1/
 | > Number of instances : 12000
 | > Max length sequence 187
 | > Min length sequence 5
 | > Avg length sequence 98.23658333333333
 | > 0 instances are ignored by min_seq_len (0)
 | > Batch group shuffling is active.
 | > Epoch 0/5000

The only changes I had made to the config.json file are in respect to the data/output paths, and number of epochs (5000 in my case). After running train.py for around 30 min, no updates are printed in the terminal. However, running `nvidia-smi` indicates that my GPUs are still under load, despite the lack of updates to the epochs in the terminal:

 Wed Jan  2 18:19:57 2019       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 396.44                 Driver Version: 396.44                    |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|===============================+======================+======================|
|   0  GeForce GTX 970     Off  | 00000000:02:00.0  On |                  N/A |
| 30%   61C    P2    52W / 163W |    594MiB /  4039MiB |      1%      Default |
+-------------------------------+----------------------+----------------------+
|   1  GeForce GTX 1080    Off  | 00000000:06:00.0 Off |                  N/A |
| 42%   63C    P2    50W / 180W |    633MiB /  8119MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
|   2  GeForce GTX 1080    Off  | 00000000:07:00.0 Off |                  N/A |
| 35%   57C    P2    50W / 200W |    575MiB /  8119MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
|   3  GeForce GTX 1080    Off  | 00000000:08:00.0 Off |                  N/A |
| 43%   66C    P2    54W / 180W |    575MiB /  8119MiB |    100%      Default |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                       GPU Memory |
|  GPU       PID   Type   Process name                             Usage      |
|=============================================================================|
|    0      1692      G   /usr/lib/xorg/Xorg                            14MiB |
|    0      4872      G   /usr/lib/xorg/Xorg                            92MiB |
|    0     10470      C   python3                                      474MiB |
|    1     10470      C   python3                                      621MiB |
|    2     10470      C   python3                                      563MiB |
|    3     10470      C   python3                                      563MiB |
+-----------------------------------------------------------------------------+

Are there additional config changes I have to make in order to see updates of where my training is currently at, or might there be an issue elsewhere?
Thanks!

**Comment by erogol** [comment]
Never experienced something like this. Some suggestion would be... 

You might like to test dataloader in isolation.

Or maybe it's a printing issue only then something about the system. Maybe you can use tee command to pass terminal output to a file. 

You can also try to reinstall pytorch using conda. 

Sorry I couldn't help much.

**Comment by fennex628** [comment]
Reinstalling PyTorch seemed to have fixed the problem for me, thanks!
